docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters/create-clusters-geopartition.md

```diff
@@ -28,6 +28,8 @@ To view your cluster tablespaces, you can enter the following command:
 SELECT * FROM pg_tablespace;
 ```
 
+Note that data placed in regions in tablespaces is not replicated to other regions, and remains subject to the fault tolerance of the cluster (Node- or Availability Zone-level).
```

Comment:
Wondering if we can rephrase "Note that data placed in regions in tablespaces" because it could be misleading. You can set up tablespaces to be multi-region if you wanted. Maybe something along the lines of "Note that data pinned to a single region via tablespaces"